app/src/main/java/net/squanchy/schedule/view/EventItemView.java

```diff
@@ -45,23 +48,24 @@ protected void onFinishInflate() {
     void updateWith(Event event) {
         timestampView.setText("12:00");         // TODO put start time 
         titleView.setText(event.title());
-        experienceLevelIconView.setExperienceLevel(event.experienceLevel());
+        if (event.experienceLevel().isPresent()) {
+            experienceLevelIconView.setExperienceLevel(event.experienceLevel().get());
+        }
 
         speakerView.setVisibility(event.speakersVisibility());
         speakerView.updateWith(createDummySpeakersList());                      // TODO use real data
     }
 
     private List<Speaker> createDummySpeakersList() {
         FirebaseSpeaker firstSpeaker = new FirebaseSpeaker();
-        firstSpeaker.avatarImageURL = String.format(Locale.US, SPEAKER_PHOTO_PATH_TEMPLATE, "squanchy.webp");
-        firstSpeaker.speakerId = 0L;
-        firstSpeaker.firstName = "Dave";
-        firstSpeaker.lastName = "Clements";
+        firstSpeaker.photo_url = String.format(Locale.US, SPEAKER_PHOTO_PATH_TEMPLATE, "squanchy.webp");
+        firstSpeaker.id = "0";
+        firstSpeaker.name = "Dave Clements";
         FirebaseSpeaker secondSpeaker = new FirebaseSpeaker();
-        secondSpeaker.avatarImageURL = String.format(Locale.US, SPEAKER_PHOTO_PATH_TEMPLATE, "squanchy.webp");
-        secondSpeaker.speakerId = 0L;
-        secondSpeaker.firstName = "Qi";
-        secondSpeaker.lastName = "Qu";
-        return Arrays.asList(Speaker.create(firstSpeaker), Speaker.create(secondSpeaker));
+        secondSpeaker.photo_url = String.format(Locale.US, SPEAKER_PHOTO_PATH_TEMPLATE, "squanchy.webp");
+        secondSpeaker.id = "0";
+        secondSpeaker.name = "Qi Qu";
+        return Arrays.asList(Speaker.create(firstSpeaker, checksum.getChecksumOf(firstSpeaker.id)),
```

Comment:
You could've just put a random number there, speakers do not need a numeric ID... which leads me onto the latter question, _why do speakers have numeric IDs?_

`/me runs to the Speaker model`